public/data/job-openings-data.json

```diff
@@ -1,36 +1,51 @@
 {
   "jobs": [
     {
-      "companyName": "SolGuruz LLP",
+      "companyName": "Metizsoft Solutions Private Limited",
```

Comment:
New object should be at the end of the file, Not replacing other entries.
Thanks